.therapist.yml

```diff
@@ -7,6 +7,14 @@ actions:
       - .venv/
       - kinto/plugins/admin/node_modules/
 
+  isort:
+    run: isort --recursive --check-only .isort.cfg {files}
+    fix: isort {files}
+    include: "*.py"
+    exclude:
+      - .venv/
+      - kinto/plugins/admin/node_modules/
```

Comment:
Shouldn't we put those in isort config instead?  Otherwise `make isort` will scan them :)